api/src/main/resources/workbench.yaml

```diff
@@ -648,9 +648,27 @@ definitions:
   SearchRequest:
     type: object
     required:
+      - include
+    properties:
+      include:
+        type: array
+        items:
+          $ref: "#/definitions/SearchGroup"
+      exclude:
+        type: array
+        items:
+          $ref: "#/definitions/SearchGroup"
+
+  SearchGroup:
```

Comment:
Would it be smart to add a description for each new element to make sure everyone understands what these represent?